### PR TITLE
Homework 1 - Kangkana 

### PR DESCRIPTION
**Pull request by kangkanak**
After opening the Read.me I started to type to edit the document but realized it was more complicated than I thought. I had to make use of the TA multiple times, it was very hard but eventually with the combination of googling and asking my classmates and the TA questions I was able to figure it out. The hardest part was incorporating a picture and the easier parts were using italics and headers once I googled R Markdown syntax reference guide. This guide included a lot of helpful information about what text to enter to get what information.

### CODE REVIEW

**Comment by jfan3** [comment]
I can't see your `.md` file. Can you commit again? Thanks!

**Comment by jfan3** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           | x             |            |
| **Coding strategy**         |           |x             |            |
| **Presentation**            |           |x              |            |
| **Achievement, creativity** |          |              |x            |
| **Ease of access**          |x           |              |            |

Remarks:
* Overall I think you touched on a lot of features that Markdown has to offer, like italics, bold, figure attaching etc. 
* However, I think you layout and content can use some extra thoughts.

**Comment by HebaAlex** [comment]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |  x            |            |
| **Ease of access**          |           |    x          |            |

**Comment by bensoltoff** [comment]
@kangkanak you did not include an update `README.md` file in your pull request. Please commit and push this file, otherwise we cannot evaluate your assignment

**Comment by JinDenguchicago** [comment]
Coding style | x |   |  
Coding strategy | x |   |  
Presentation |  x|  |  
Achievement, creativity |   | x |  
Ease of access |   | x

**Comment by jin-deng** [comment]
You did not include you README.rmd. Probably you forgot to push it.

**Comment by kangkanak** [comment]
> @kangkanak you did not include an update `README.md` file in your pull request. Please commit and push this file, otherwise we cannot evaluate your assignment

Sorry about that I just submitted it now, I was confused on what Fiona meant earlier!

**Comment by kangkanak** [comment]
> I can't see your `.md` file. Can you commit again? Thanks!

I just submitted it I'm so sorry I didn't realize what you meant and then got your comment so I thought I was all good!

**Comment by jin-deng** [comment]
Coding style: satisfactory
Coding strategy: needs work
Presentation: excellent
Achievement, creativity: excellent
Ease of access: excellent

-Elaborate on above, especially for "needs work."
Your picture does not show up when I clicked the link for the picture.

-Some specific praise?
You are doing a good job of putting a youtube link.

-Something I learned?
The sublist is a good way to show what you enjoy.

-Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
Setting up GitHub is not easy, especially committing and pushing. I guess it is a difficulty for you as well. Once we successfully do it, we will be fine in the future.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # hw01
```
Comment (jfan3) [inline README.md:1]: Redundant

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+Here are some things I enjoy doing!
+
+
+ * Dancing
+ * Traveling
+ * Reading
+ * Listening to Rap/Hip-Hop/R&B
+ 
+ 
+ Here is a link to my favorite **Frank Ocean** song:
```
Comment (jfan3) [inline README.md:22]: You can do it as `[Here](https://www.youtube.com/watch?v=r4l9bFqgMaQ) is a link to my favorite **Frank Ocean** song:`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+ 
+ 
+ [LINK](https://www.youtube.com/watch?v=r4l9bFqgMaQ)
+ 
+
+### Here is a picture from Garfield Conservatory, one of my favorite places in Chicago!
```
Comment (jfan3) [inline README.md:28]: The sectioning here seems a bit odd. You can use `##` instead.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+
+
+![](IMG-1787.JPG)
```
Comment (jfan3) [inline README.md:31]: Here in the `[]` you can add the alternative words in case the picture didn't appear.

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |            |x              |            |
| **Coding strategy**         |            |x               |            |
| **Presentation**            |           |x              |            |
| **Achievement, creativity** |           |x              |            |
| **Ease of access**          |            |              |x            |

Remarks:
* Overall you have hit the major features that markdown has to offer. 
* You can ignore/not include the redundant files in the directory. 
* The overall layout and content can use a bit more work.

**Pull request closed without merging**